Serializable/SerializableTests/Tests/DecodableTests.swift

```diff
@@ -19,8 +19,8 @@ class DecodableTests: XCTestCase {
 		do {
 			if let path = Bundle(for: type(of: self)).path(forResource: "DecodableTest", ofType: "json"), let data = try? Data(contentsOf: URL(fileURLWithPath: path)) {
 				bridgedDictionary = try JSONSerialization.jsonObject(with: data, options: .allowFragments) as? NSDictionary
-				testModels = DecodableModel.array(bridgedDictionary.object(forKey:"models") as AnyObject?)
-				testNonArrayModels = DecodableModel.array(bridgedDictionary.object(forKey:"model") as AnyObject?)
+				testModels = DecodableModel.array(bridgedDictionary.object(forKey:"models"))
```

Comment:
Cast not needed as the array class method has changed to expect Any instead of AnyObject